core/keyboard_nav/ast_node.js

```diff
@@ -491,26 +490,6 @@ Blockly.ASTNode.prototype.findFirstFieldOrInput_ = function(block) {
   return null;
 };
 
-/**
- * Walk backwards from the given block up through the stack of blocks to find
- * the top block of the sub stack. If we are nested in a statement input only
- * find the top most nested block. Do not go all the way to the top of the
```

Comment:
This says it doesn't go all the way to the top of the stack, unlike `getRootBlock`.

Reply:
Yup, caught that and then fell down the rabbit hole of trying to figure out what the neighbouring `getFirstStatementConnection` was for.